src/librustdoc/clean/types.rs

```diff
@@ -810,13 +811,15 @@ impl AttributesExt for [ast::Attribute] {
                 doc_cfg
                     .filter_map(|attr| Cfg::parse(attr.meta_item()?).ok())
                     .fold(Cfg::True, |cfg, new_cfg| cfg & new_cfg)
-            } else {
+            } else if doc_auto_cfg_active {
```

Comment:
This only has an effect if `doc_cfg` is also active. I think it makes sense to enable them independently, but if you don't want to work on that right now, please make it a hard error to enable doc_auto_cfg without also enabling `doc_cfg`.

Reply:
Good point. In addition to that, it's pretty easy to only enable `doc_auto_cfg` so it'd be a shame not to do it!